### PR TITLE
Add Unit tests

### LINKED ISSUE #382: improve unit test coverage
**Is your feature request related to a problem?/Why is this needed**

**Describe the solution you'd like in detail**

improve UT coverage including
 - controllerserver
 - nodeserver
 - identityserver
 - version

https://coveralls.io/github/kubernetes-sigs/azuredisk-csi-driver?branch=master

image

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Sakuralbj**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind test

**What this PR does / why we need it**:
Unit tests
**Which issue(s) this PR fixes**:

Fixes #382 

**Special notes for your reviewer**:
First try. I'll do controllerserver and nodeserver quickly.

**Release note**:
```

```

### CODE REVIEW

**Comment by Sakuralbj** [comment]
> @Sakuralbj: The following test **failed**, say `/retest` to rerun all failed tests:
> 
> Test name	Commit	Details	Rerun command
> pull-azuredisk-csi-driver-verify	[23d5bd5](https://github.com/kubernetes-sigs/azuredisk-csi-driver/commit/23d5bd5f0e017489a42c779f222d28e848b479f0)	[link](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_azuredisk-csi-driver/419/pull-azuredisk-csi-driver-verify/1268056885550911488)	`/test pull-azuredisk-csi-driver-verify`
> [Full PR test history](https://prow.k8s.io/pr-history?org=kubernetes-sigs&repo=azuredisk-csi-driver&pr=419). [Your PR dashboard](https://prow.k8s.io/pr?query=is%3Apr%20state%3Aopen%20author%3Sakuralbj). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/sig-testing/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/kubernetes-sigs/azuredisk-csi-driver/issues?q=is:issue+is:open) when you hit one in your PR.
> 
> Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md). If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).

/retest

**Comment by andyzhangx** [comment]
```
Boilerplate header is wrong for: /home/prow/go/src/sigs.k8s.io/azuredisk-csi-driver/hack/../pkg/azuredisk/identityserver_test.go
Boilerplate header is wrong for: /home/prow/go/src/sigs.k8s.io/azuredisk-csi-driver/hack/../pkg/azuredisk/version_test.go
make: *** [Makefile:52: verify] Error 1
```

run `make verify` in your local env, and you will get same error. The header of the new file is wrong.

File: pkg/azuredisk/version_test.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package azuredisk
```
Comment (andyzhangx) [inline pkg/azuredisk/version_test.go:1]: add a header

**Pull request closed without merging**